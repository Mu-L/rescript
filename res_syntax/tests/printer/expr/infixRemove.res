let plus = (x, y) => x + y
let minus = (x, y) => x - y

@@infix.remove(("ğŸ˜€", "plus"))
@@infix.remove(("ğŸ’©ğŸ’©", "minus"))

let q = 3 ğŸ˜€ 4 ğŸ’©ğŸ’© 5
