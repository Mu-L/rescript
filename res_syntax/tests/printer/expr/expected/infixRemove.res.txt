let plus = (x, y) => x + y
let minus = (x, y) => x - y

@@infix.remove(("ğŸ˜€", "plus"))
@@infix.remove(("ğŸ’©ğŸ’©", "minus"))

let q = minus(plus(3, 4), 5)
